### PR TITLE
kwargs for lut params in inversion & doc review 

### PR DESCRIPTION
**Pull request by vincelhx**

### CODE REVIEW

**Comment by vincelhx** [comment]
did make html

File: src/xsarsea/windspeed/utils.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import os
+from pathlib import Path
```
Comment (agrouaze) [inline src/xsarsea/windspeed/utils.py:2]: curiosity question: why `pathlib` rather than `os` ? Is there a benefit?

**Review by agrouaze — CHANGES_REQUESTED** [review]
just a suggestion of change for import utils

File: src/xsarsea/windspeed/utils.py (hunk 1 above)
Comment (vincelhx) [reply src/xsarsea/windspeed/utils.py:2]: no i guess its the same, i changed it

File: src/xsarsea/windspeed/models.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 import netCDF4
 import logging
 import pandas as pd
-
+from .utils import _load_config_luts
```
[the remark this answers is not in the document]
Comment (vincelhx) [reply src/xsarsea/windspeed/models.py:11]: i will try to modify on the next release but not this one

[the remark this answers is not in the document]
Comment (agrouaze) [reply src/xsarsea/windspeed/models.py:11]: Ok for me, just create an issue. Nice work of review of the doc!

**Review by agrouaze — APPROVED** [review]

**Pull request merged**